src/Channel.java

```diff
@@ -0,0 +1,98 @@
+/**
+ * Created by kenshoo-linux on 14/07/16.
+ */
+/**
+ * Created by ken-linux2 on 12/07/16.
+ */
+public class Channel {
+
+    private String name;
+    private String channel_campaign_id;
```

Comment:
Please change the camel case: channelCampaignId.
This applies to all field, please use the idea to rename the field and the getter/setter (so you'll learn how to use it)